src/validate.js

```diff
@@ -2649,6 +2649,24 @@ function F_VALIDATE(disabledHL) {
 				}
 			}// global speed limit check
 
+			// GROUP isDrivable
+			// on named segment, make sure theres a city on primary or alt
+			if (!cityLen && streetLen && RT_RAMP !== roadType && RT_FREEWAY !== roadType) {
```

Comment:
nit: for the sake of performance, we should start with the cheapest checks first, so it seems `isLimitOk()` should be somewhere here, at the end of the condition.

Reply:
Since 54 & 55 both use this, it would have to be (isLimitOk(54) || isLimitOk(55))

Reply:
limit checks added on first condition